src/node.h

```diff
@@ -305,6 +305,14 @@ NODE_EXTERN v8::Local<v8::Context> NewContext(
     v8::Local<v8::ObjectTemplate> object_template =
         v8::Local<v8::ObjectTemplate>());
 
+// Runs Node.js-specific tweaks on an already constructed context
+// This is not dead code, it is exposed for embedders so that they
+// can construct their own context and still use it in a node Environment.
```

Comment:
I think for public APIs the “no dead code” disclaimer is not necessary :)

What does the `Maybe` in `MaybeInitializeContext` refer to? There *is* a `TODO` comment inside the method about the fact that it *should* return a `v8::MaybeLocal`. But I’m not sure if that’s what you mean? Either way, it would be good to address that `TODO` comment here as well.

Reply:
I called it maybe out of personal convention, it can return an empty `Local` which is unusual and normally unexpected (which is why I think the `TODO` comment is there).  Happy to try address that in this PR if that would make the signature a bit nicer 👍